### DISCUSSION TITLE
Forschungsergebnisse und Diplomarbeiten / Research results and theses

### DISCUSSION THREAD

**Discussion by christianheimann** [e-id-admin/governance-sounding-board#46] [Ideas]
Hast du eine Diplomarbeit gemacht rund um das Thema E-ID oder SSI-Ökosystem? Dann gerne hier im Thread vermerken und wenn möglich Link zum Download angeben!
__________
Tu as réalisé un travail de diplôme sur le sujet de l'e-ID ou de l'écosystème SSI ? Dans ce cas, n'hésite pas à le mentionner dans ce thread de discussion et indique si possible le lien de téléchargement !
__________
Avete fatto una tesi sull'E-ID o sull'ecosistema SSI? Se sì, segnalatelo qui nel thread e, se possibile, fornite un link per il download!
__________
Have you done a thesis on the topic of E-ID or the SSI ecosystem? If so, please note it here in the thread and, if possible, provide a link to the download!

**Comment by cybee42** [comment]
Two master theses have been completed at the CYD Campus. You can find the reports and corresponding POC code in these repositories:

- [KERI Under Scrutiny: Use-Case Study for the Swiss e-ID](https://github.com/luffa99/KERI-Under-Scrutiny-Master-Thesis/)
- [Scalable and Privacy-Preserving Revocation of Verifiable Credentials](https://github.com/alecolo129/eid-revocation-rs)

**Comment by cybee42** [comment]
Another two master theses were recently completed. One is performing a detailed security analysis of KERI. The other is exploring generic zero-knowledge proofs (ZKP) for verifiable credentials.

- [A Security Analysis of KERI](https://www.research-collection.ethz.ch/handle/20.500.11850/735690)
- [General-Purpose Zero-Knowledge Proofs for Verifiable Credentials](https://github.com/mombelld/general-purpose-zkps-vcs)
